src/locale/languages/de.js

```diff
@@ -0,0 +1,347 @@
+// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
+// Link: https://github.com/gitpoint/git-point/pull/260
+
+export const de = {
+  auth: {
+    login: {
+      connectingToGitHub: 'Verbinde mit GitHub...',
+      preparingGitPoint: 'Bereite GitPoint vor...',
+      cancel: 'ABBRECHEN',
+      welcomeTitle: 'Willkommen bei GitPoint',
+      welcomeMessage:
+        'Der umfangreichste GitHub Client der zu 100% kostenlos ist',
+      notificationsTitle: 'Verwalte Benachrichtigungen',
+      notificationsMessage: 'Finde und Verwalte alle deine Benachrichtigungen',
+      reposTitle: 'Repositorien und Benutzer',
```

Comment:
würde man eher `Repositories` sagen?

Reply:
@staabm Habe da drüber viel nachgedacht, Repositorien ist der korrekte deutche Begriff obwohl es niemand sagt, lässt sich im Nachhinein ja noch anpassen. 

Jetzt bitte wieder englisch, damit uns alle verstehen.

Reply:
discussing german terms in english is strange and english speaking persons could not help in this discussion nevertheless..

Reply:
`Repositorien` sounds a bit scientific, never read it anywhere to be honest (except glossaries)

Reply:
@staabm Yes, its the official translation, no one would ever say it 😆